modules/svg/register_types.cpp

```diff
@@ -30,10 +30,8 @@
 
 #include "register_types.h"
 
-#include "core/os/os.h"
 #include "image_loader_svg.h"
 
-#include "thirdparty/thorvg/inc/config.h"
```

Comment:
Normally a library is responsible for including its own `config.h` where relevant, we should haven't to do it ourselves before including the main entry point. Should be double checked in case ThorVG doesn't do this properly. It seems to include it itself in two places:
```
src/lib/tvgLzw.cpp
56:#include "config.h"

src/lib/tvgCommon.h
25:#include "config.h"
```

Reply:
Seems work removed in modules/svg/register_types.cpp.